content/en/docs/instrumentation/python/exporters.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Exporters
+weight: 4
+---
+
+In order to visualize and analyze your traces and metrics, you will need to
+export them to a backend.
+
+## Console exporter
+
+The console exporter is useful for development and debugging tasks, and is the
+simplest to set up.
+
+```python
+from opentelemetry import trace
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor, ConsoleSpanExporter
+from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+
+# Service name is required for most backends
+resource = Resource(attributes={
+    "service.name": "your-service-name"
+})
+
+provider = TracerProvider(resource=resource)
+processor = BatchSpanProcessor(ConsoleSpanExporter())
+provider.add_span_processor(processor)
+trace.set_tracer_provider(provider)
+
+# Merrily go about tracing!
+```
+
+## OTLP endpoint or Collector
+
+To send data to an OTLP endpoint or the [OpenTelemetry
+Collector](/docs/collector/getting-started/), you'll want to configure an OTLP
+exporter that sends to your endpoint.
+
+First, install an OTLP exporter:
+
+```shell
+$ pip install opentelemetry-exporter-otlp-proto-http
```

Comment:
Just a clarification here regarding command code blocks. There are two choices:

- Either the language is set to `shell` and no prompt is used, or
- The language is set to `console` and prompt(s) are used.

The latter has the advantage of formatting commands and their output differently.

Reply:
Yep, about to do that - I realized I should have just written `console`. Early morning funsies 🙂